### PR TITLE
add weakly up to leader actions list

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/common/cluster.md [hunk 1]
```diff
@@ -296,6 +296,7 @@ The `leader` has the following duties:
 
  * shifting members in and out of the cluster
     * joining -> up
+    * weakly up -> up (if enabled); *no convergence is required for this leader action to be performed*
```
Comment (patriknw) [inline akka-docs/src/main/paradox/common/cluster.md]: it's enabled by default in 2.5 so not sure if that () is needed

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/common/cluster.md (hunk 1 above)
Comment (ktoso) [reply akka-docs/src/main/paradox/common/cluster.md]: Good catch, thanks! :)

**Pull request merged**